ui/selectors/shared.ts

```diff
@@ -0,0 +1,2 @@
+export const EMPTY_ARRAY = [];
+export const EMPTY_OBJECT = {};
```

Comment:
Is there a reason to avoid `Object.freeze()` here? Aside from general safety reasons, for the selectors in `ui/selectors/assets.ts` this would be a breaking change.

```suggestion
export const EMPTY_ARRAY = Object.freeze([]);
export const EMPTY_OBJECT = Object.freeze({});
```